tests/TestCase/I18n/DateTest.php

```diff
@@ -112,6 +112,16 @@ public function testI18nFormat($class)
 
         $result = $time->i18nFormat(\IntlDateFormatter::FULL, null, 'es-ES');
         $this->assertContains('14 de enero de 2010', $result, 'Default locale should not be used');
+
+        $time->setToStringTimezone('Europe/Brussels');
+        $result = $time->i18nFormat('HH:mm:ss');
+        $expected = '14:59:28';
+        $this->assertEquals($expected, $result);
```

Comment:
Actually this raises another issue. Date and FrozenDate need to ignore the timezone setting, otherwise it can corrupt date output!